status/types.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package neo.fs.v2.status;
+
+option go_package = "github.com/nspcc-dev/neofs-api-go/v2/status/grpc;status";
+option csharp_namespace = "Neo.FileStorage.API.Status";
+
+// Declares the general format of the status returns of the NeoFS RPC protocol.
+// Status is present in all response messages. Each RPC of NeoFS protocol
+// describes the possible outcomes and details of the operation.
+//
+// Each status is assigned a one-to-one numeric code. Any unique result of an
+// operation in NeoFS is unambiguously associated with the code value.
+//
+// Numerical set of codes is split into 1024-element sections. An enumeration
+// is defined for each section. Values can be referred to in the following ways:
+//
+// * numerical value ranging from 0 to 4,294,967,295 (global code);
+//
+// * values from enumeration (local code). The formula for the ratio of the
+//   local code (`L`) of a defined section (`S`) to the global one (`G`):
+//   `G = 1024 * S + L`.
+//
+// All outcomes are divided into successful and failed, which corresponds
+// to the success or failure of the operation. The definition of success
+// follows from the semantics of RPC and the description of its purpose.
+// The server must not attach code that is the opposite of outcome type.
+//
+// See the set of return codes in the description for calls.
+//
+// Each status can carry developer-facing error message. It should be human
+// readable text in English. The server should not transmit (and the client
+// should not expect) useful information in the message. Field `details`
+// should make the return more detailed.
+message Status {
+    // The status code.
```

Comment:
No periods there and down below if comment is one sentence long.

Reply:
Fixed.